crates/next-binding/src/lib.rs

```diff
@@ -1,53 +1,34 @@
 #[cfg(feature = "__swc")]
 pub mod swc {
     #[cfg(feature = "__swc_core")]
-    pub mod core {
-        pub use swc_core::*;
-    }
+    pub use swc_core as core;
 
     #[cfg(feature = "__swc_custom_transform")]
     pub mod custom_transform {
+        #[cfg(feature = "__swc_transform_modularize_imports")]
+        pub use modularize_imports;
         #[cfg(feature = "__swc_transform_styled_components")]
-        pub mod styled_components {
-            pub use styled_components::*;
-        }
+        pub use styled_components;
         #[cfg(feature = "__swc_transform_styled_jsx")]
-        pub mod styled_jsx {
-            pub use styled_jsx::*;
-        }
+        pub use styled_jsx;
         #[cfg(feature = "__swc_transform_emotion")]
-        pub mod emotion {
-            pub use swc_emotion::*;
-        }
-        #[cfg(feature = "__swc_transform_modularize_imports")]
-        pub mod modularize_imports {
-            pub use modularize_imports::*;
-        }
+        pub use swc_emotion as emotion;
     }
 
     #[cfg(feature = "testing")]
-    pub mod testing {
-        pub use testing::*;
-    }
+    pub use testing;
 }
 
 #[cfg(feature = "__turbo")]
 pub mod turbo {
-    #[cfg(feature = "__feature_next_dev_server")]
-    pub mod dev_server {
-        pub use next_dev::*;
-    }
-
-    #[cfg(feature = "__feature_node_file_trace")]
-    pub mod node_file_trace {
-        pub use node_file_trace::*;
-    }
+    #[cfg(feature = "__turbo_next_dev_server")]
+    pub use next_dev as next_dev_server;
```

Comment:
I just realized this is probably my mistake to rename imports, the intention is transparently reexports without rename. Mind to change to export it as original name?

Reply:
But `swc::core` stays?

Reply:
done

Reply:
that I need to think bit, I felt bit redundant to make `swc::swc_core` 🤔

Reply:
true